src/models/user.py

```diff
@@ -30,11 +31,12 @@ def fields(self):
             'date_registered' : (b.is_date_string,),
             'last_sign_in' : (b.is_date_string,),
             'courses' : (b.is_list,),
-            'departments' : (b.is_list,),
+            'departments' : (),
             'created_surveys' : (b.is_list,),
             'unanswered_surveys' : (b.is_list,),
             'answered_surveys' : (b.is_list,),
             'answers' : (b.is_list,),
+            'primary_affiliation' : (),
```

Comment:
Remember: later on down the line, people will need to be able to flag themselves as *both* student and faculty. Although we are making it dichotomous now, we should be sure that we don't mistakenly deny ourselves now. 

This should be a list, and it should be treated like a list.